foursight_core/react/api/aws_s3.py

```diff
@@ -13,7 +13,7 @@ class AwsS3:
     def get_buckets(cls) -> list:
         results = []
         try:
-            s3 = boto3.resource("s3")
+            s3 = boto_s3_resource()
```

Comment:
Whether this is a good rewrite depends on the result of the other long comment I made about what paradigm to use. I won't mark these in detail.

Reply:
Yeah, these are gone now, using the monkey patching thing for this.